.github/workflows/check-runtimes-template-spec-chain.yml

```diff
@@ -0,0 +1,135 @@
+name: check-runtimes-template-spec-chain.yml
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+  merge_group:
+  workflow_dispatch:
+
+jobs:
+  preflight:
+    uses: ./.github/workflows/reusable-preflight.yml
```

Comment:
I don't think we need this. Seeing the preflight used specifically when needing to run this on custom runners for certain cases (like release activities). We should be fine with `ubuntu-latest` for the jobs bellow.